ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/Translation.hs

```diff
@@ -87,8 +87,10 @@ tests = testGroup "UpdateTablesOnEraTransition"
                           byronToShelleyLedgerStateTranslation
                           byronUtxosAreInsertsInShelleyUtxoDiff
                           (\st -> cover 50 (      nonEmptyUtxosByron st) "UTxO set is not empty"
-                                -- TODO: #4473 we should test with empyt UTxO!
-                                -- . cover 1  (not $ nonEmptyUtxosByron st) "UTxO set is empty"
+                                . cover 1  (not $ nonEmptyUtxosByron st) "UTxO set is empty"
```

Comment:
Adding this makes the tests very slow in some cases, because the probability of generating an empty UTxO is still relatively low. Do we still want to add this `cover`?

Reply:
Seems to be solved by requiring `0.1` coverage instead of `1`.